### PR TITLE
Improves pair check

### PR DESCRIPTION
**Pull request by TobsCore**
Instead of rotating the board multiple times, pairs are checked in both a vertical and horizontal fashion. Improves code readability and performance (as if it was needed, but still..)

### CODE REVIEW

**Comment by mevdschee** [comment]
Okay, this is a possible improvement, but why did you leave the rotateBoard function in the code?

**Comment by TobsCore** [comment]
The `rotateBoard` function is still used by `moveLeft`, `moveDown` etc.

**Comment by mevdschee** [comment]
>The rotateBoard function is still used by moveLeft, moveDown etc.

Then duplication of this piece of code and not reusing the rotation seems like an arbitrary change to me, not necessarily an improvement.

File: 2048.c [hunk 1]
```diff
@@ -213,12 +222,8 @@ bool gameEnded(uint8_t board[SIZE][SIZE]) {
 	bool ended = true;
```
Comment (mevdschee) [inline 2048.c]: no longer used?

File: 2048.c [hunk 2]
```diff
@@ -213,12 +222,8 @@ bool gameEnded(uint8_t board[SIZE][SIZE]) {
...
 	if (countEmpty(board)>0) return false;
 	if (findPairDown(board)) return false;
-	rotateBoard(board);
-	if (findPairDown(board)) ended = false;
-	rotateBoard(board);
-	rotateBoard(board);
-	rotateBoard(board);
-	return ended;
+	if (findPairDown(board)) return false;
```
Comment (mevdschee) [inline 2048.c]: did you mean "findPairSide"?

**Pull request merged**